crypt.go

```diff
@@ -584,12 +583,7 @@ func ActivateVolumeWithMultipleTPMSealedKeys(tpm *TPMConnection, volumeName, sou
 		return false, err
 	}
 
-	if err := activateWithTPMKeys(tpm, volumeName, sourceDevicePath, keyPaths, passphraseReader, options.PassphraseTries, activateOptions, options.KeyringPrefix); err != nil {
-		errs, ok := err.(activateWithTPMKeysErrors)
-		if !ok {
-			return false, xerrors.Errorf("unexpected error: %w", err)
-		}
-
+	if success, errs := activateWithTPMKeys(tpm, volumeName, sourceDevicePath, keyPaths, passphraseReader, options.PassphraseTries, activateOptions, options.KeyringPrefix); !success {
```

Comment:
I think the change here is good but the function return still does not feel super natural/go-ish. But no idea what else could be done to improve so +1 :)